splinterd/man/splinterd.1.md

```diff
@@ -237,29 +245,83 @@ self-signed certificates and keys (which can be generated by the
 Development](https://github.com/Cargill/splinter-docs/blob/master/docs/howto/generating_insecure_certificates_for_development.md)"
 in the Splinter documentation.
 
+SPLINTER DIRECTORY PATHS
+========================
+
+Several Splinter directories have the following default locations:
+
+* Splinter configuration directory: `/etc/splinter`
+
+* State directory: `/var/lib/splinter/`
+
+* TLS certificate directory: `/etc/splinter/certs/`
+
+For the configuration and certificate directories, the directory paths can be
+changed individually with a `splinterd` option, a setting in a TOML config file,
+or an environment variable. (The state directory location is controlled only by
```

Comment:
The state directory config/option should be added.

Reply:
Will make a story as part of stabilization clean up